src/Compilers/CSharp/Test/Syntax/Parsing/SeparatedSyntaxListParsingTests.cs

```diff
@@ -205,7 +206,7 @@ class C
 }
 
 class M<,> { }
-");
+", options: TestOptions.Regular.WithTuplesFeature());
```

Comment:
I pulled the parser tests that were modified due to tuples and pulled their original code back in, with modification to run with C# 6 language version.